share/github-backup-utils/ghe-restore-audit-log

```diff
@@ -22,55 +22,54 @@ setup(){
   ghe_remote_version_required "$GHE_HOSTNAME"
 }
 
-# Check whether the snapshot comes from an instance
-# where the MySQL import was complete
-is_import_complete(){
-  test -e "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/audit-log/mysql-import-complete"
+# Check whether the snapshot contains audit logs that
+# were taken from Elasticsearch
+es_data_available(){
+  ls -A "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT"/audit-log/*.size >/dev/null 2>&1
 }
 
-# Check whether the snapshot was taken on an instance
-# where MySQL audit logs were enabled
-mysql_restored_enabled(){
-  test -e "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT/audit-log-mysql"
+# Check whether the snapshot contains audit logs that
+# were taken from MySQL
+mysql_dump_available(){
+  ls -A "$GHE_DATA_DIR/$GHE_RESTORE_SNAPSHOT"/audit-log-mysql/20*.gz >/dev/null 2>&1
 }
 
-remove_complete_flag(){
-  ghe_verbose "Setting instance(s) as pending for audit log import to MySQL"
-  ghe-ssh "$GHE_HOSTNAME" -- "sudo rm -rf $GHE_REMOTE_ROOT_DIR/data/user/common/audit-log-import/complete" 1>&3 2>&3
+# Check whether the remote host is running a version where the MySQL backend
+# is supported, i.e: < 2.19
+is_mysql_supported(){
+   [ "$(version $GHE_REMOTE_VERSION)" -lt "$(version 2.19.0)" ]
+}
+
+set_remote_flag(){
```

Comment:
minor: what do you think of naming these more specific to audit-log, since it's mostly related to audit log import settings.
```suggestion
set_remote_audit_log_import_flag(){
```

Reply:
didn't realize this was in it's own `audit-log` named file, so modified the above suggestion to be more explicit what we're setting.

Reply:
That was a helper function that is used by the two below. I added more doc to the functions to clarify their usage.